modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,206 @@ To view a list of global command options that you can use with any `virtctl` com
+
+|`virtctl vmexport download <vmexport_name> --output=<output_file> --volume=<volume_name>`
+|Download the volumes defined in a `VirtualMachineExport` CR.
+
+* `--output` specifies the file format. Example: `disk.img.gz`.
+* `--volume` specifies the volume to download. This flag is optional if only one volume is available.
+
+Optional:
+
+* `--keep-vme` retains the `VirtualMachineExport` CR after download. The default behavior is to delete the `VirtualMachineExport` CR after download.
+* `--insecure` enables an insecure HTTP connection.
+
+|`virtctl vmexport download <vmexport_name> --<vm\|snapshot\|pvc>=<object_name> --output=<output_file>`
+|Create a `VirtualMachineExport` CR and then download the volumes defined in the CR.
+|===
+
+[id='vm-memory-dump_{context}']
+== VM memory dump
+
+You can use the `virtctl memory-dump` command to output a virtual machine (VM) memory dump on a PVC. You can specify an existing PVC or use the `--create-claim` flag to create a new PVC.
+
+.Prerequisites
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --proxy-only=true <vmi-name>`
-|Display the port number and connect manually to the virtual machine instance by using any viewer through the VNC connection.
+* The PVC volume mode must be `FileSystem`.
+* The PVC must be large enough to contain the memory dump.
++
+The formula for calculating the PVC size is `(VMMemorySize + 100Mi) * FileSystemOverhead`, where `100Mi` is the memory dump overhead.
 
-|`virtctl vnc --kubeconfig=$KUBECONFIG --port=<port-number> <vmi-name>`
-|Specify a port number to run the proxy on the specified port, if that port is available. If a port number is not specified, the proxy runs on a random port.
+* You must enable the hot plug feature gate in the `HyperConverged` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc patch hco kubevirt-hyperconverged -n openshift-cnv \
+  --type json -p '[{"op": "add", "path": "/spec/featureGates", \
+  "value": "HotplugVolumes"}]'
+----
+
+.Downloading the memory dump
+
+// Workaround for 4.12
+Download the memory dump by using the `virtctl vmexport download` command:
+
+[source,terminal]
+----
+$ virtctl vmexport download <vmexport_name> \
+  --vm=<vm_name> --output=<output_file>
+----
+
+.`virtctl memory-dump` commands
+[width="100%",cols="1a,2a",options="header"]
+|===
+|Command |Description
+|`virtctl memory-dump get <vm_name> --claim-name=<pvc_name>`
+|Save the memory dump of a VM on a PVC. The memory dump status is displayed in the `status` section of the `VirtualMachine` resource.
+
+Optional:
+
+* `--create-claim` creates a new PVC with the appropriate size. This flag has the following options:
+
+** `--storage-class=<storage_class>` specifies a storage class.
+** `--access-mode=<access_mode>` specifies an access mode, for example, `ReadOnlyMany` or `ReadWriteMany`.
+
+// * `--output=<output_file>` gets a memory dump and downloads it to a specified output file, for example, `--output=memoryDump.dump.gz`.
```

Comment:
this flag is relevant only for the memory dump download command that we said is added only in 4.13

Reply:
That's why I commented it out.